Connect4.py

```diff
@@ -1,7 +1,92 @@
 GRID_WIDTH = 7
 GRID_HEIGHT = 6
 
-grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+class Board:
+
+    def __init__(self):
+        self.grid = [["_" for i in range(GRID_WIDTH)] for j in range(GRID_HEIGHT)]
+
+    def draw(self):
+        print("\n  0    1    2    3    4    5    6")
+        for row in self.grid:
+            print(row)
+
+    def drop(self, col):
+        col = int(col)
```

Comment:
make these functions assume their inputs are ints and do any conversion in the input phase.